packages/block-library/src/search/edit.js

```diff
@@ -257,6 +257,7 @@ export default function SearchEdit( {
 						type="button"
 						className={ buttonClasses }
 						style={ buttonStyles }
+						aria-label={ label ? label : __( 'Search' ) }
```

Comment:
@Sisanu Probably going to want to make this buttonLabel. Need to figure out how to remove all the HTML tags though. Not sure if we have package for it or not.

```
import { __unstableStripHTML } from '@wordpress/dom';
```

That may be safe to use. I saw another PR get merged using it. Still I am unsure why it is marked unstable.

Reply:
@alexstine, are you aiming for something like this? As far as I understand we have a `buttonText` attribute.
```aria-label={ buttonText ? stripHTML(buttonText) : __( 'Search' ) }```

Reply:
@Sisanu Yes.

Reply:
Committed, please review.